Pocket/Pocket/Protocols/PocketProtocol.swift

```diff
@@ -8,15 +8,14 @@
 
 import Foundation
 
-public typealias CompletionHandler = (_:Pocket) -> Void
+public typealias PocketInitHandler = (_:PocketProtocol) -> Void
 
-public protocol Pocket: AnyObject {
+public protocol PocketProtocol: AnyObject {
```

Comment:
Do not suffix a protocol with the word `protocol`. 

From [Apple's Guidelines](https://swift.org/documentation/api-design-guidelines/#use-terminology-well)

```
Protocols that describe what something is should read as nouns (e.g. Collection).

Protocols that describe a capability should be named using the suffixes able, ible, or ing (e.g. Equatable, ProgressReporting).
```

This protocol should probably be broken up into two protocol in two files:

- `PocketWallet` with a `create` function and an `import` function
- `PocketNetworking` with a `createTransaction`function and a `createQuery` function